packages/story-editor/src/components/checklist/checkpointContext/checkpointContext.js

```diff
@@ -71,14 +71,9 @@ function ChecklistCheckpointProvider({ children }) {
   );
 
   const [highPriorityCount, setHighPriorityCount] = useState();
-  const [reviewDialogRequested, setReviewDialogRequested] = useState();
+  const [reviewChecklistRequested, setReviewChecklistRequested] = useState();
```

Comment:
Nit: feels like this name is a little confusing to me. This keeps track of if the checklist should show the contents, right? Maybe to make it a little easier to follow it could be named to something like `showAllChecklistItems` or something 🤔 

But not a big deal. Works well as it is.